pkg/apis/serving/register.go

```diff
@@ -162,6 +180,30 @@ var (
 		QueueSidecarResourcePercentageAnnotationKey,
 		"queue.sidecar." + GroupName + "/resourcePercentage",
 	}
+	QueueSidecarCPUResourceRequestAnnotation = kmap.KeyPriority{
+		QueueSidecarCPUResourceRequestAnnotationKey,
+		"queue.sidecar." + GroupName + "/CPUResourceRequest",
```

Comment:
I'm wondering if we should simply call this 'sidecar.serving.knative.dev' - 'queue' in the name is sorta an implementation detail

Reply:
Right now this is the only one sidecar we tune here, I am concerned that if we make this more generic someone might try use it with Istio or some other sidecar they use assuming Knative manages that too.